### PR TITLE
manifests: update apiserver config CRD with EncryptionType spec

### PR DESCRIPTION
**Pull request by sttts**
Follow-up of openshift/api#453.

glide.yaml had been broken in https://github.com/openshift/cluster-config-operator/pull/94.

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -1,9 +1,15 @@
 package: github.com/openshift/cluster-config-operator
 import:
-  - package: k8s.io/klog
-    version: v1.0.0
+- package: k8s.io/klog
+  version: v1.0.0
```
Comment (mfojtik) [inline glide.yaml]: @sttts can you bump this to version v0.4.0 ? (it is what origin use and I think 1.16.x)

Comment (sttts) [reply glide.yaml]: done

**Comment by enj** [comment]
LGTM, verify failure looks real.

**Pull request merged**